plugin/azure/README.md

```diff
@@ -0,0 +1,87 @@
+# azure
+
+## Name
+
+*azure* - enables serving zone data from Microsoft Azure DNS service.
+
+## Description
+
+The azure plugin is useful for serving zones from resource record
+sets in Microsoft Azure DNS. The `azure` plugin can be used when coredns is deployed on Azure or elsewhere.
+
+## Syntax
+
+~~~ txt
+azure [resource-group:dns-zone] {
+    [azure_auth_location <path to azure_auth_location>]
+    [azure_tenant_id <azure_tenant_id>]
+    [azure_client_id <azure_client_id>]
+    [azure_client_secret <azure_client_secret>]
+    [azure_subscription_id <azure_subscription_id>]
+    upstream
```

Comment:
`upstream` is a keyword is not needed anymore. Just make an upstream.New() in the code; that get's used is there is a forward (or something else to resolve) in this coredns

Reply:
gotcha, removed.